tools/chrome-debug-extension/README.md

```diff
@@ -0,0 +1,75 @@
+
+You have several options for configurations:
+1. Use a preset configuration
+   - Ideal if your team already has one built that you can reuse - can be highly customized to your telemetry schema
+1. Use the default configuration
+   - Most suitable for situations where you are debugging the integration of Application Insights into your web application
+1. Use a local custom configuration
+   - Good for customizing the table layout to best match your telemetry schema
+   - Easily and rapidly iterate on your configuration right within the tool without having to recapture session data
+       - Click the gear icon to go to the configuration page to modify your custom configuration, then save it to see the results
+   - The custom configuration is stored in local storage, so you won't lose it unless you uninstall the plugin
+   - You can copy out the configuration and share it with coworkers - best for early prototyping or if you aren't able to add a preset configuration
+1. Create a new preset configuration
+   - To add your configuration to the preset configuration list, you'll need to check in a change to [`Configuration.types.ts`](https://github.com/microsoft/ApplicationInsights-JS/blob/master/tools/chrome-debug-extension/src/configuration/Configuration.types.ts) to add your configuration's URL to the list. 
+   - Once your configuration is in the list, you will need to follow your team's process for updating the configuration file, since it is hosted in a team-specific location. It may be in a repo where you have to check in changes to it, or it may be on a OneDrive share that you can update directly.
+
+## Contributing to the tool
+
+1. Please see the general information about contributing to this repository [here](https://github.com/microsoft/ApplicationInsights-JS/tree/master#contributing)
+1. To specifically work on this tool, follow this dev inner loop:
+   1. Startup:
+      1. Go into Chrome or Edge, go to the `Manage Extensions` page (edge://extensions or chrome://extensions/), and `Remove` the tool if you already have it installed from somewhere other than your enlistment
+      1. In the repo root, run `rush install`
+      1. Run `rush build --to @microsoft/applicationinsights-chrome-debug-extension`
+      1. In the `Manage Extensions` page, click `Load unpacked` and browse to your local `tools/chrome-debug-extension/browser` folder and click `Select Folder`
+      1. You can now use your locally built version of the tool in your browser, and use the F12 developer tools to debug the popup
+   1. Making incremental changes:
+      1. Save your change
+      1. Run `rush build --only @microsoft/applicationinsights-chrome-debug-extension`
+      1. If you only modified the popup window, just press F5 in the popup window
+      1. If you modified any of the content/background code, click `Reload` for the plugin in the `Manage Extensions` page
```

Comment:
same here, numbering is messed up

Reply:
For markdown you don't have to number each item incrementally